scripts/rmc.py

```diff
@@ -93,19 +93,86 @@ def symbol_table_to_gotoc(json_filename, cbmc_filename, verbose=False, keep_temp
     return process.returncode
 
 
-def run_cbmc(cbmc_filename, cbmc_args, verbose=False, quiet=False):
+def run_cbmc(cbmc_filename, cbmc_args, verbose=False, quiet=False, output_to=None):
     cbmc_cmd = ["cbmc"] + cbmc_args + [cbmc_filename]
     process = subprocess.Popen(cbmc_cmd, universal_newlines=True,
                                stdout=subprocess.PIPE,
                                stderr=subprocess.STDOUT)
-    if not quiet:
+    if output_to != None:
+        with open(output_to, "w") as f:
+            for line in iter(process.stdout.readline, ""):
```

Comment:
I would use process.stdout.splitlines() to split the lines in an OS independent fashion. Also in the not quiet case below.

Reply:
Fixed